src/api/services/timecardService.spec.js

```diff
@@ -143,4 +145,34 @@ describe('Timecard Service', () => {
       }
     });
   });
+
+  describe('deleteTimeEntry', () => {
+    it('should return 200 on success', async () => {
+      api.delete.mockImplementation(() => Promise.resolve({ status: 200 }));
+
+      const response = await deleteTimeEntry(123, TENANT_ID_PARAM);
+
+      expect(api.delete).toHaveBeenCalledWith(
+        expect.stringContaining('resources/time-entry/123'),
+        TENANT_ID_PARAM
+      );
+      expect(response.status).toBeDefined();
+    });
+
+    it('should throw a useful error containing throwing service and function', async () => {
```

Comment:
I think this test name/description should specify a 'when' condition similar to how we specify on the success tests.
e.g '_should throw a useful error containing throwing service and function_ **when an error occurs**'
I can see that it's just a copy paste situation as we've all named the others like that.

Reply:
I've updated this test and the others in the file too

Reply:
nice one thanks :)